need a code review, new to process.

jomivega opened the pull request:
Hello all, I had not done a pull request before, so none of my code had been seen by y'all. Would love the feedback and a code review!

jomivega committed:
need a code review, new to process.

samschaevitz commented on Reminder/app/src/main/java/com/reminder/app/LoginActivity.java:
```diff
@@ -0,0 +1,137 @@
+        context = this.getApplicationContext();
+        findViewById(R.id.sign_in_button).setOnClickListener(this);
+        map = ((MapFragment)getFragmentManager().findFragmentById(R.id.map)).getMap();
+        LatLng test = new LatLng(-33.867, 151.206);
```
Add a "TODO(jordan): ..." to remove this test when you have real data?

samschaevitz commented on Reminder/app/src/main/java/com/reminder/app/LoginActivity.java:
```diff
@@ -0,0 +1,137 @@
+        setContentView(R.layout.login);
+        context = this.getApplicationContext();
+        findViewById(R.id.sign_in_button).setOnClickListener(this);
+        map = ((MapFragment)getFragmentManager().findFragmentById(R.id.map)).getMap();
```
can you split this line into a few lines to make it easier to read?

samschaevitz commented on Reminder/app/src/main/java/com/reminder/app/LoginActivity.java:
```diff
@@ -0,0 +1,137 @@
+        LatLng test = new LatLng(-33.867, 151.206);
+        map.setMyLocationEnabled(true);
+        map.moveCamera(CameraUpdateFactory.newLatLngZoom(test, 13));
+        map.addMarker(new MarkerOptions()
```
Maybe all of these map init calls can be broken out into a helper function?

samschaevitz commented on Reminder/app/src/main/java/com/reminder/app/LoginActivity.java:
```diff
@@ -0,0 +1,137 @@
+            resolveSignInError();
+        }
+    }
+    @Override
```
Let's always do this (add @Override to other class methods, please). Keeps inheritance easier to reason about.

samschaevitz commented on Reminder/app/src/main/java/com/reminder/app/LoginActivity.java:
```diff
@@ -0,0 +1,137 @@
+    public void onConnected(Bundle connectionHint) {
+        mSignInClicked = false;
+        if (Plus.PeopleApi.getCurrentPerson(mGoogleApiClient) != null) {
+            currentPerson = Plus.PeopleApi.getCurrentPerson(mGoogleApiClient);
```
Let's split the Person setup and the Email setup out into helpers for readability.

samschaevitz commented:
Couple of comments; didn't really look at the XML files but so long as you add a comment saying it works (TESTED=...whatever you did to test this code...) in the commit, I'm fine with it.

jomivega committed:
Code refactoring and Standardize UI Style update

jomivega commented:
Cleaned up some of the code and worked on a standardized UI. Here are some images of the UI. Do you all like the profile picture in the navigation bar? If so, on the left or right side? Also, I changed the navigation icon to white, and positioned it. I changed the font of the items in the drawer as well. Let me know what you think! 
![screenshot_2014-06-27-00-59-21](https://cloud.githubusercontent.com/assets/6498651/3409147/2faa5310-fdd5-11e3-9a4b-eb633d973741.png)
![screenshot_2014-06-27-00-58-29](https://cloud.githubusercontent.com/assets/6498651/3409149/32421072-fdd5-11e3-9e6e-8a0f0bf5e817.png)

![screenshot_2014-06-27-00-58-15](https://cloud.githubusercontent.com/assets/6498651/3409139/270c302a-fdd5-11e3-8a7d-c133379a15ae.png)

samschaevitz commented:
I vote for right side; it feels more balanced. Digging the font and color schema! Maybe there could be sub menus in the left bar for "Upcoming" and "Done" reminders?

jomivega commented:
Ooh yes I like that Idea. Should I work on that after I merge what I have here?

samschaevitz commented:
SGTM (sounds good to me)

On Fri Jun 27 2014 at 11:20:50 AM, jomivega notifications@github.com
wrote:

> Ooh yes I like that Idea. Should I work on that after I merge what I have
> here?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/GoogleJump/PinkFairyArmadillos/pull/3#issuecomment-47383367
> .

jomivega commented:
Awesome. Just waiting for an okay from either Kylie or Jooeun.

samschaevitz commented:
Can you call one of them out as a reviewer on this pull request? That way
it'll be clearer who needs to own the review.

On Fri Jun 27 2014 at 11:35:31 AM, jomivega notifications@github.com
wrote:

> Awesome. Just waiting for an okay from either Kylie or Jooeun.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/GoogleJump/PinkFairyArmadillos/pull/3#issuecomment-47384987
> .

JooeunLim commented:
I like the right side as well, and the upcoming/done list idea too. Should
the add reminder page have the same green or a different color?

On Friday, June 27, 2014, Samantha Schaevitz notifications@github.com
wrote:

> Can you call one of them out as a reviewer on this pull request? That way
> it'll be clearer who needs to own the review.
> 
> On Fri Jun 27 2014 at 11:35:31 AM, jomivega <notifications@github.com
> <javascript:_e(%7B%7D,'cvml','notifications@github.com');>>
> wrote:
> 
> > Awesome. Just waiting for an okay from either Kylie or Jooeun.
> > 
> > —
> > Reply to this email directly or view it on GitHub
> > <
> > https://github.com/GoogleJump/PinkFairyArmadillos/pull/3#issuecomment-47384987>
> > 
> > .
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/GoogleJump/PinkFairyArmadillos/pull/3#issuecomment-47385297
> .

jomivega commented:
What do you mean Jooeun? Like should you add the top bar and drawer? Or? Jooeun can you also be the reviewer for this pull request?

JooeunLim commented:
No, I mean just the color scheme in general. I was thinking the add
reminder page  would just have a back button or something to go back home
and not have the sidebar. And yup!

On Friday, June 27, 2014, jomivega notifications@github.com wrote:

> What do you mean Jooeun? Like should you add the top bar and drawer? Or?
> Jooeun can you also be the reviewer for this pull request?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/GoogleJump/PinkFairyArmadillos/pull/3#issuecomment-47385901
> .

jomivega commented:
If you create the add reminder page using the same color scheme, it will be perfect. Contact me, or look through the colors.xml to get the value of the green. No need for a back button.

jomivega committed:
Adding Kylies layout and java files to standarized UI.

jomivega commented:
In my latest commit, I have added Kylie's UI from the master branch. 
Need to talk you Kylie about the layout so that it works with the container/standardized UI I have set up. 

Also, for all that are interested, I have ran the go backend through golint and made changes accordingly. You can see the changes in this commit if you would like. 

Sam, let me know what you think about the backend code.

jomivega committed:
Kylie's layout working with standardized UI container.

jomivega committed:
Creating sub menus for drawer

jomivega commented:
Updated the UI and added kylie's UI to the standardized UI containter. Let me know your thoughts about the action bar and submenues.
![screenshot_2014-06-27-18-23-41](https://cloud.githubusercontent.com/assets/6498651/3419573/703aaa86-fe6d-11e3-98f3-8132aa871b43.png)
![screenshot_2014-06-27-18-39-12](https://cloud.githubusercontent.com/assets/6498651/3419574/7344ed9a-fe6d-11e3-9a03-7a470d6ad072.png)
![screenshot_2014-06-27-18-39-18](https://cloud.githubusercontent.com/assets/6498651/3419577/76e03d88-fe6d-11e3-89b0-ef5824f2b50f.png)

jomivega commented:
_Ignore the white spots_

jomivega committed:
fixing project to work with kylie

bumbledog commented:
Still not compiling correctly... Jooen, could you see if it runs on your computer?

bumbledog commented:
Here's the error I keep getting.
2032-2032/com.reminder.app E/﹕ Can't open file for reading
This happens whenever I try to login to google plus on the app via an emulator.

bumbledog commented:
I'm 90% sure it's a connection problem with the backend. Whenever I try to step through, I'm getting a lot of connection errors.

bumbledog commented:
Oops, I did not mean to close this...

jomivega committed:
Json Parsing into "Reminders" table and add reminder button.

jomivega committed:
Expandable ListView implementation

jomivega committed:
Expandable ListView working with single reminders and group of reminders. Timed notifications.

jomivega committed:
Swipe and notification bug fix

jomivega committed:
Adding location notifications and UI changes

jomivega committed:
adding UI

jomivega committed:
Adding UI changes that didnt commit correctly

jomivega committed:
adding REST calls

Commit:
Text and Urgency support for AddReminder Page.
Date and Location in the works.
Changed min API version.

Commit:
Merge branch 'Jordan' of https://github.com/GoogleJump/PinkFairyArmadillos into Jordan

Conflicts:
	Reminder/app/src/main/java/com/reminder/app/MainActivity.java
	Reminder/app/src/main/java/com/reminder/app/RESTClient.java

Commit:
Adding basic functionality to Coordinates
